assignments/tasks.go

```diff
@@ -75,6 +75,11 @@ func handleTasks(ctx context.Context, db database.Database, sc scm.SCM, course *
 		return err
 	}
 
+	// TODO(Espeland): This will create all tasks in the course creator's name.
```

Comment:
Not sure I see a problem with this? That said, the new GitHub app that @0xf8f8ff is developing in the context of the improved security architecture -- for that we may need to reconsider this.